app/forms/schools/add_participants/wizard_steps/yourself_step.rb

```diff
@@ -15,8 +15,6 @@ def next_step
             else
               :cannot_add_manual_transfer
             end
-          elsif wizard.needs_to_confirm_start_term?
-            :start_term
           elsif wizard.needs_to_choose_a_mentor?
```

Comment:
```suggestion
          elsif wizard.needs_to_confirm_start_term? && !Cohort.within_next_registration_period?
            :cannot_add_registration_not_yet_open
          elsif wizard.needs_to_confirm_start_term? && wizard.need_training_setup?
            :need_training_setup
          elsif wizard.needs_to_choose_a_mentor?
```

Reply:
Same as above

Reply:
I don't understand why we have added logic and 2 different steps here - could you explain please?

Reply:
Thanks for clarifying, this has been amended to allow both outcomes to be visited, I think this is where we perhaps need additional test coverage.